### DISCUSSION TITLE
Assets extra configuration prompts

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxtrdev/nuxtr-vscode#132] [Questions]
Middleware don't ask if it's default or global.
Server api/routes don't ask the HTTP method.
Plugin don't ask if it's .client, .server or both (same with pages, components, layouts...)

### Env
Windows 11, WSL (Ubuntu), Nuxtr 0.2.16

**Comment by adhamfarrag** [comment]
@angelhdzmultimedia Hi 👋

Middleware don't ask if it's default of global because you can just add `.global` prefix when you create one either using context menu or command pallete.
Server api/routes don't ask the HTTP method because the goal was to offer an entry point for the file, nothing else. And soon you should be able to create templates for everything. 
Plugin don't ask if it's `.client` because again you can just add it. For most users it will be much easier than just adding an extra step for you. 

Nuxtr's goal is to make you focus on delivery and part of it just adding a systematic way with optional steps for the sake of it that not all users need. Let me know what you think :)
